post/controller.js

```diff
@@ -5,29 +5,25 @@ const express = require('express');
 const router = express.Router();
 const urlencodedParser = bodyParser.urlencoded({ extended: false });
 
-router.get(['/', '/posts'], (req, res) => {
+router.get('/posts', (req, res) => {
   postService.getAllPosts().then((posts) => {
-    res.render('posts', { posts });
+    res.json(posts);
```

Comment:
So it looks like the data returned here is of the form:
```
{
	"_id": "5aae75e0f7ca0a5257e07cdd",
	"title": "First Post",
	"body": "Content",
	"__v": 0
}
```

Is that the intention? Do you want to return the `_id` field and the `__v` field?